show_cli/show_paths.go

```diff
@@ -0,0 +1,17 @@
+package show_cli
```

Comment:
let's be consistent here, if we use show_client, let's use show_client in all places. cli is not equal to client.

Reply:
Done